data_rentgen/server/api/v1/router/auth.py

```diff
@@ -28,3 +44,33 @@ async def token(
         login=form_data.username,
     )
     return AuthTokenSchema.model_validate(user_token)
+
+
+@router.get("/callback")
+async def auth_callback(
+    request: Request,
+    state: str,
+    code: str,
+    auth_provider: Annotated[KeycloakAuthProvider, Depends(Stub(AuthProvider))],
+):
+    state = base64.b64decode(state.encode("utf-8"))  # type: ignore[assignment]
+    original_url = state.decode("utf-8")  # type: ignore[attr-defined]
+
+    if not original_url:
+        raise HTTPException(status_code=HTTPStatus.BAD_REQUEST, detail="Invalid original_url")
```

Comment:
`original_url` is not used anymore, why is it required?

Reply:
Actually I'm think we should use it latter.
I am think about additional header with user `location` and return it to frontend.

Reply:
Ok